src/chainConstants.ts

```diff
@@ -8,12 +9,26 @@ export const VOLTA_DOMAIN_NOTIFER_ADDRESS =
 export const VOLTA_ENS_REGISTRY_ADDRESS =
   '0xd7CeF70Ba7efc2035256d828d5287e2D285CD1ac';
 export const VOLTA_CLAIM_MANAGER_ADDRESS =
-  '0x561785174DF7f564f2591bA52B253c0F663427aB';
+  '0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031';
 export const VOLTA_IDENTITY_MANAGER_ADDRESS =
   '0x84d0c7284A869213CB047595d34d6044d9a7E14A';
 export const VOLTA_REWARD_POOL_ADDRESS =
   '0x6d832DDd386F1dFA53d5b0cACcFF509f95D345Ca';
 export const VOLTA_RESOLVER_V2_ADDRESS =
   '0xcf72f16Ab886776232bea2fcf3689761a0b74EfE';
 
+// EWC chain
 export const EWC_CHAIN_ID = 246;
+export const EWC_CLAIM_MANAGER_ADDRESS =
+  '0x5339adE9332A604A1c957B9bC1C6eee0Bcf7a031';
```

Comment:
I think this address is wrong
```suggestion
  '0x23b026631A6f265d17CFee8aa6ced1B244f3920C';
```